espnet2/bin/asr_inference.py

```diff
@@ -107,6 +111,19 @@ def __init__(
             token_list=token_list,
             pre_beam_score_key=None if ctc_weight == 1.0 else "full",
         )
+
```

Comment:
- We need some flag to enable/disable transducer for multi task training.
- If constructing BeamSearchTransducer, stop to construct BeamSearch.

Reply:
I didn't know what you wanted for decoding so I let it like that.
For now I think we should use `asr_model.rnnt_decoder` to control and construct either one of them based on existence. Is that ok?